pkg/kratos/handlers.go

```diff
@@ -30,14 +30,15 @@ func (a *API) RegisterEndpoints(mux *chi.Mux) {
 // TODO: Validate response when server error handling is implemented
 func (a *API) handleCreateFlow(w http.ResponseWriter, r *http.Request) {
 	q := r.URL.Query()
+	loginChallenge := q.Get("login_challenge")
 
 	// We try to see if the user is logged in, because if they are the CreateBrowserLoginFlow
 	// call will return an empty response
 	// TODO: We need to send a different content-type to CreateBrowserLoginFlow in order
 	// to avoid this bug.
 	session, _, _ := a.service.CheckSession(context.Background(), r.Cookies())
 	if session != nil {
-		redirectTo, cookies, err := a.service.AcceptLoginRequest(context.Background(), session.Identity.Id, q.Get("login_challenge"))
+		redirectTo, cookies, err := a.service.AcceptLoginRequest(context.Background(), session.Identity.Id, loginChallenge)
```

Comment:
might wanna do the same change here as well 

https://github.com/canonical/identity-platform-login-ui/pull/208/files#diff-12151c8a9259cc64c5b82a9a51ed3eb6367a4850988ecd45cb323b4e086c708cL70
https://github.com/canonical/identity-platform-login-ui/pull/208/files#diff-12151c8a9259cc64c5b82a9a51ed3eb6367a4850988ecd45cb323b4e086c708cL75